docs-conceptual/azurermps-5.7.0/install-azurerm-ps.md

```diff
@@ -38,161 +40,137 @@ PowerShellGet 1.6.0   C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1
 
-## Reporting issues and feedback
+## Uninstall the Azure PowerShell module
 
-If you encounter any bugs with the tool, file an issue in the
-[Issues](https://github.com/Azure/azure-powershell/issues) section of our GitHub repo. To provide
-feedback from the command line, use the `Send-Feedback` cmdlet.
+To uninstall the Azure PowerShell module, use the [Uninstall-Module](/powershell/module/powershellget/uninstall-module) cmdlet. To uninstall 
+the most recent version of the Azure PowerShell on your system:
 
-## Frequently asked questions
+```powershell
+Uninstall-Module -Name AzureRM
+```
 
-### How to get PowerShellGet
+You can also delete a single version:
 
-|Scenario|Install instructions|
-|---|---|
-|Windows 10<br/>Windows Server 2016|Built into Windows Management Framework (WMF) 5.0 included in the OS|
-|I want to upgrade to PowerShell 5|<ol><li>[Install the latest version of WMF](https://www.microsoft.com/en-us/download/details.aspx?id=54616)</li><li>Run the following command:<br/>```Install-Module PowerShellGet -Force```</li></ol>|
-|I am running on a version of Windows with PowerShell 3 or PowerShell 4|<ol><il>[Get the PackageManagement modules](http://go.microsoft.com/fwlink/?LinkID=746217)</il><li>Run the following command:<br/>```Install-Module PowerShellGet -Force```</li></ol>|
-
-<a id="helpmechoose"></a>
-### Checking the version of Azure PowerShell
+```powershell
+Uninstall-Module -Name AzureRM -RequiredVersion 6.1.0
+```
 
-Although we encourage you to upgrade to the latest version as early as possible, several versions
-of Azure PowerShell are supported. To determine the version of Azure PowerShell you have installed,
-run `Get-Module AzureRM` from your command line.
+Or completely remove Azure PowerShell from your system.
 
 ```powershell
-Get-Module AzureRM -ListAvailable | Select-Object -Property Name,Version,Path
+Uninstall-Module -Name AzureRM -AllVersions
 ```
 
-### Support for classic deployment methods
-
-If you have deployments that use the classic deployment model you can install the Service
-Management version of Azure PowerShell. For more information, see [Install the Azure PowerShell
-Service Management module](/powershell/azure/servicemanagement/install-azure-ps). The Azure and AzureRM modules share
-common dependencies. If you use both the Azure and AzureRM modules, you should install the same
-version of each package.
+To uninstall all versions of Azure PowerShell except for the latest, run the following
+command:
 
-### <a id="update-azps"></a>Updating to a new version of Azure PowerShell
+```powershell
+Get-InstalledModule -Name AzureRM -AllVersions |
+    Where-Object -Property Version -ne $(Get-InstalledModule -Name AzureRM).Version |
+    ForEach-Object {Uninstall-Module -Name $_.Name -RequiredVersion $_.Version}
+```
 
-If you have a previous version of Azure PowerShell installed that includes the Service Management
```

Comment:
This is a very common scenario and we got lots of comments about it. That is why it is included. If you remove then then you need to add -AllowClobber to the install example above.

Reply:
Is this still a common scenario? How many users are updating from a version containing ASM to the ARM model these days?

Will create/add in "Troubleshooting" to help address issues like this one.

Reply:
Interestingly I don't see any errors where an install fails when using `Install-Module` after installing from MSI. I tried the following, and they all succeeded:

```
Install-Module -Name AzureRM
Install-Module -Name AzureRM -RequiredVersion 5.7.0
Install-Module -Name Azure
Install-Module -Name Azure -RequiredVersion 4.0.0
```

Is there a reproducible case that you know of? Maybe the AzPS team silently fixed the MSI install issue, or ASM was updated so that there were no longer conflicts.

Reply:
I don't know if this has changed. What version of PowerShell and PowerShellGet did you test with?

Reply:
Tested with PowerShell `5.1.17134.48` and PowerShellGet `1.0.0.1`. I noticed the minimum reqs say that 1.1.0.0 is required - so not sure why this worked?

Reply:
PowerShellGet 1.0.0.1 does not have the -AllowClobber parameter. So if you have another version you will get an error and won't be able to work around it by using the clobber option.